tests/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
 RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
     ln -s /cache/images /build/images && \
+    ln -s /cache/images /build/virt-builder && \
```

Comment:
> The file names are incremented with a number here:

Ah, ok. So as there's no debian-8.1 (for example) I suppose that these images never actually change (which is a pity, as it will need an ever-increasing dist upgrade for each usage, but that's not under our control).